### DISCUSSION TITLE
Direct pip installation

### DISCUSSION THREAD

**Discussion by terryfrankcombe** [mir-group/allegro#36] [General]
Hi folks

I appreciate that on the main page the installation instructions say work from source from the git repo.  However, I did try straight `pip install allegro` and seemed to make lots of progress.  Overall it failed, with `httptools`, `greenlet` and `uvloop` failing to compile.

When I did clone and install from source `pip install .` worked fine.  Interestingly, there was no mention of those previously-failed packages being required.

Is this expected?  Is there a dependency list somewhere that needs updating?

Ciao
Terry

**Comment by Linux-cpp-lisp** [comment]
Hi Terry,

The "Allegro" [package on PyPI](https://pypi.org/project/Allegro/) has no relation to this project;  I should definitely put a warning about that in the README.  At present, Allegro (unlike the `nequip` package) has not been pushed to PyPI;  I'll add it to the README when that happens.

**Reply by terryfrankcombe** [reply]
Wow.  Yes, that's obviously a problem!

**Reply by Linux-cpp-lisp** [reply]
I've added a warning to the README, sorry about that!
